pynecone/pc.py

```diff
@@ -118,23 +118,28 @@ def run(
     frontend_cmd = backend_cmd = None
     if env == constants.Env.DEV:
         frontend_cmd, backend_cmd = exec.run_frontend, exec.run_backend
-    if env == constants.Env.PROD:
+    if env == constants.Env.PROD or env == constants.Env.PREVIEW:
         frontend_cmd, backend_cmd = exec.run_frontend_prod, exec.run_backend_prod
     assert frontend_cmd and backend_cmd, "Invalid env"
 
     # Post a telemetry event.
     telemetry.send(f"run-{env.value}", get_config().telemetry_enabled)
 
     # Run the frontend and backend.
-    # try:
     if backend:
         threading.Thread(
             target=backend_cmd, args=(app.__name__, backend_port, loglevel)
         ).start()
     if frontend:
-        threading.Thread(
-            target=frontend_cmd, args=(app.app, Path.cwd(), frontend_port)
-        ).start()
+        if env == constants.Env.DEV:
```

Comment:
We can simplify to something like

```python
args = [app.app, Path.cwd(), frontend_port, loglevel]
if env != constants.Env.DEV:
    args.append(env)
threading.Thread(target=frontend_cmd, args=args)
```

Also  what's the reason we don't want to pass in the env in dev mode?

Reply:
There was no need to pass it in dev mode, I pass it into prod mode because there are two options prod, preview